packages/alchemy/src/middleware/types/simulate-uo/index.ts

```diff
@@ -0,0 +1,18 @@
+import type { UserOperationStruct } from "@alchemy/aa-core";
+import type { Address, Hash } from "viem";
+import type {
+  SimulateAssetChange,
+  SimulateAssetChangesError,
+} from "./interfaces.js";
+
+export type SimulateUserOperationAssetChangesRequest = [
+  UserOperationStruct,
+  entryPoint: Address,
+  blockNumber?: Hash
+];
+
+export type SimulateUserOperationAssetChangesResponse = {
+  changes: SimulateAssetChange[];
+  gasUsed?: string;
```

Comment:
nit: gasUsed is not available for v0 and might never be included, we may include other fields in the future.

Reply:
awesome thanks for checking